File: ATM2.py
# cook your dish here
for i in range(int(input())):
    n,k=map(int,input().split())
    ar=list(map(int,input().split()))
    s=""
    for i in range(len(ar)):
        if k>=ar[i]:
            k=k-ar[i]
            s=s+"1"
        else:
            s=s+"0"
    print(s)
    

File: ATM2.cpp
#include <iostream>
using namespace std;

int main() {
    // your code goes here
    int t;
    cin>>t;
    while(t--)
    {
        int n,k;
        cin>>n>>k;
        int ar[n];
        string s;
        for(int i=0;i<n;i++){
            cin>>ar[i];
            if(ar[i]<=k){
                k=k-ar[i];
                s=s+'1';
            }
            else{
                s=s+'0';
            }
        }
        cout<<s<<endl;
    }
	return 0;
}
